app/assets/javascripts/modules/testsByWeekWidget.js

```diff
@@ -0,0 +1,85 @@
+angular.module('probedock.testsByWeekWidget', [ 'probedock.api' ])
+  .directive('testsByWeekWidget', function() {
+    return {
+      restrict: 'E',
+      controller: 'TestsByWeekChartCtrl',
+      templateUrl: '/templates/tests-by-week-widget.html',
+      scope: {
+        organization: '=',
+        project: '='
```

Comment:
There should be a parameter for the number of weeks. Please display more than 10 weeks on the dashboard.